### ISSUE TITLE
Spinner doesn't clear completely when done

### ISSUE THREAD

**Issue by maxkorp** [google/zx#478]
## Sample Code
```
await spinner('Truncating jobs tables...', async () => {
  await $`sleep 5`;
});
console.log('jobs/job_queues truncated');

await spinner('Scaling deployments back up...', async () => {
    await $`sleep 5`;
});
console.log('Deployments scaled back up');
```

## Expected Behavior
With the above code, I would expect to see 
```
[*] Truncating jobs tables...
```
then, once that spinner completes
```
jobs/job_queues truncated
```
then 
```
jobs/job_queues truncated
[*] Scaling deployments back up...
```
then when it completes
```
jobs/job_queues truncated
Deployments scaled back up
```

## Actual Behavior
```
[*] Truncating jobs tables...
```
then, once that spinner completes
```
jobs/job_queues truncateds...
```
then 
```
jobs/job_queues truncateds...
[*] Scaling deployments back up...
```
then when it completes
```
jobs/job_queues truncateds...
Deployments scaled back upck up...
```
Basically it appears to not be flushing out what it wrote to stdout/err properly. 

## Specifications

- Version: 7.0.7
- Platform: MacOS, node 16

**Comment by antonmedv** [comment]
I see. Last output of spinner should be bunch of white spaces to clear previous message.

**Comment by antonmedv** [comment]
Fixed in b6e9ca9f710309e214e27dbbfacd7dffceaeaec1
